manim/mobject/svg/text_mobject.py

```diff
@@ -717,7 +717,7 @@ def full2short(self, config):
         text2weight -> t2w
         """
         for kwargs in [config, self.CONFIG]:
-            if kwargs.__contains__("text2color"):
+            if "text2color" in kwargs:
                 kwargs["t2c"] = kwargs.pop("text2color")
             if kwargs.__contains__("text2font"):
```

Comment:
Please change all of the others as well :)

Reply:
will do soon.